src/main/flight/mixer.c

```diff
@@ -121,6 +121,7 @@ static FAST_RAM_ZERO_INIT float motorMixRange;
 
 float FAST_RAM_ZERO_INIT motor[MAX_SUPPORTED_MOTORS];
 float motor_disarmed[MAX_SUPPORTED_MOTORS];
+static FAST_RAM_ZERO_INIT float throttlehpf;
```

Comment:
Better `throttleHpf`.

Reply:
In mixer.c JoeLucid named the internal throttle highness `throttlehpf`.  I didn't change any of that.

When passing to pid.c I used ThrottleHpf as the 'carrier', and inside pid.c I used throttleHpf.

I don't know about it being a good idea just changing the capitalisation... but that's what I did  :-)

Happy to modify as appropriate.

Reply:
Naming convention for Betaflight is to use camelCase for variables, so `throttleHpf` fits with the convention, yes please.

Reply:
Fixed.  Thanks for the explanation.